pkg/cache/v2/snapshot.go

```diff
@@ -40,6 +55,15 @@ func IndexResourcesByName(items []types.ResourceWithTtl) map[string]types.Resour
 	return indexed
 }
 
+// IndexRawResourcesByName creates a map from the resource name to the resource.
+func IndexRawResourcesByName(items []types.Resource) map[string]types.Resource {
```

Comment:
probably won't need this if you switch to ResourceWithTTL

Reply:
Cool I'll pull it out